dev/Repeater/APITests/Common/CollectionChangeEventArgsConverters.cs

```diff
@@ -87,6 +87,9 @@ public static NotifyCollectionChangedEventArgs ConvertToDataSourceChangedEventAr
                         newArgs = new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Replace, addedItems, removedItems, oldStartingIndex);
                     }
                     break;
+                case NotifyCollectionChangedAction.Move:
+                    newArgs = new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Move, null, newStartingIndex, oldStartingIndex);
```

Comment:
Should we pass the moved item to the collection changed args? After all this would be the only operation where we are not pass the element(s).

Reply:
Per https://docs.microsoft.com/en-us/dotnet/api/system.collections.specialized.notifycollectionchangedeventargs.-ctor?view=netcore-3.1#System_Collections_Specialized_NotifyCollectionChangedEventArgs__ctor_System_Collections_Specialized_NotifyCollectionChangedAction_System_Object_System_Int32_System_Int32_, this constructor that takes a single element as a parameter seems like the right way to express a single item move operation. To clarify, this null is equivalent to each of the nulls that are Added to the lists in the other cases, it doesn't represent a List<object> whose value is null.

Reply:
Now that you bring it up it is worth pointing out that this change only adds support for single item move., I'll mention it in the description. I didn't add support for multi item move because it isn't supported by ObservableCollection, but it would be a nice follow up change.

Reply:
Oh yes, you are right. Interesting. Maybe this is really something we need to follow up on. While ObservableCollection does not support multi item move, neither does it support multi item add or multi item remove. Maybe we could do this in this PR? Of course only if you are comfortable with that, otherwise maybe create a tracking issue for that?

I am not sure how easy this would be to implement, I would hope that it's just adopting the same pattern as for add and remove. Testing this might be a bit more difficult thought.

Reply:
ObservableCollection does not support it, but a custom collection could be built where multiple items can be moved right ?

Reply:
Indeed, a custom collection can definitely implement it. I changed the test collections so they implement multi-item move.